server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/redirect/NotifyMailetsMessageTest.java

```diff
@@ -68,8 +72,7 @@ public void generateMessageShouldReturnTheMessageWhenSimpleMimeMessage() throws
         assertThat(generateMessage).isEqualTo("my message\n" +
                 "\n" +
                 "Message details:\n" +
-                "  MAIL FROM: user@james.org\n" +
-                "  Size: -1 B\n");
+                "  MAIL FROM: user@james.org\n");
```

Comment:
all these fixes demonstrate that asserting too much is a pain

Reply:
Fixed